website_portal_purchase/views/website.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+
+    <template id="assets_frontend" inherit_id="website.assets_frontend" name="Website Portal Sale">
+      <xpath expr="." position="inside">
+        <link rel='stylesheet' href='/website_portal_purchase/static/src/less/website_portal_purchase.less'/>
```

Comment:
I'd say this is not less bus just plain CSS. Please rename the file extension and this link.